src/base.iced

```diff
@@ -173,6 +178,16 @@ class Base
 
   #------
 
+  _v_include_full_pgp_hash : -> false
+  _v_pgp_km_to_hash : () -> @km()
+  _v_pgp_hash_dest : (body) -> body.key
+
+  full_pgp_sha256: () ->
+    if '_cached_pgp_sha256' of @ then @_cached_pgp_sha256
```

Comment:
I want to return the cached value even if it's null. Maybe `typeof @ _cached_pgp_sha256 isnt 'undefined'` would be better?